components/Animate/AnimateInView.tsx

```diff
@@ -45,7 +45,7 @@ export const AnimateInView = ({
         duration,
         ease: 'easeOut',
       }}
-      initial="hidden"
+      initial={beforeAnimationState}
```

Comment:
Updated AnimateInView wrapper. This still works as intended on live, but when this component is used with a newer version of Framer Motion, it doesn't work correctly. This get rid of the conflict and should be this way from the beginning - I thought this looked sus at one point and always meant to come back to do it properly even though it was still working.

Reply:
The reason I found the bug was that Rebecca is using my AnimateInView component on Centennial which is on React 19/Next 15 and the latest version of Framer Motion (now called just Motion). She was wondering why the reduced motion setting didn't work for her site, but still works for Momentum. I came back to look at this and found that it's fixed if I make this change. @sherakama